packages/vue/src/components/field/tests/field.test.vue

```diff
@@ -1,11 +1,12 @@
 <script setup lang="ts">
 import { Field } from '../..'
+const model = defineModel()
```

Comment:
I'm not 100% sure this is the correct way to use the `defineModel()` helper. Please ignore if I'm wrong here 😄

Reply:
For this purpose, it's to emit the model up a second level for this test file